src/eos_stream.c

```diff
@@ -293,6 +292,7 @@ static void processEosioNewAccountAction(txProcessingContext_t *context) {
         PRINTF("No delays allowed");
         THROW(EXCEPTION);
     }
+    UNUSED(read); // Remove clang scan-build deadcode.DeadStores warning
```

Comment:
Here we can drop the received `read`, pass by ref `&size` is the thing we care about. Suggest
- drop `read` in line 290
- remove `UNUSED(read)` line 295